python-avocado.spec

```diff
@@ -367,6 +367,7 @@ examples of how to write tests on your own.
 %changelog
 * Tue Aug 22 2017 Cleber Rosa <cleber@redhat.com> - 53.0-1
 - Use variable name for configuration dir
+- Clean up old changelog entries
```

Comment:
Don't know the practice here, is this allowed? Are there any rules regarding the length of changelog?